### ISSUE TITLE
How do i move from mydomain1 to mydomain2

### ISSUE THREAD

**Issue by bobbybark** [DanielnetoDotCom/YouPHPTube#1002]
My YouPHPTube installed and working, How do i move from mydomain1 to mydomain2

**Comment by DanielnetoDotCom** [comment]
Try this https://tutorials.youphptube.com/video/fix-youphptube-url-after-move-server-location-and-ip

**Comment by siniztr** [comment]
Hey,
I have a similar question. I am busy moving my streaming site from a test environment to a real environment. I downloaded all of the files from server1 (test environment) onto server2, and did a MySQL export and imported it onto the new site. I watched the video above, and made the following changes:
 $global['webSiteRootURL'] = server1 became server2 (using full FQDN).
Everything else remained the same. I even made sure to name the database and the user for the database the same thing as on the test environment. 

When I go to the server2 address, I see the framework, but it says that there are no videos. I am also not able to log in with the 3 users that I have. Admin, my own and a fellow tester.
